Cargo.toml

```
@@ -45,6 +46,9 @@ thread_local = "1.1.7"
 mockall = "0.13.0"
 core_affinity = "0.8.1"
 hex = "0.4.3"
+reqwest = { version = "0.12.7", features = ["blocking"] }
```

The blocking feature isn't needed anymore, is it?
```suggestion
reqwest = { version = "0.12.7"}
```